etc/make-lint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [[ $# -ne 0 ]]; then
+    bold "error expected 0 arguments, got $#!"
+    exit 1
+fi
+
+exit_code=0
+
+echo "Linting with ruff . . ."
+ruff check setup.py tests/*.py src/libsemigroups_pybind11/*.py docs/source/*.py || ((exit_code = 1))
```

Comment:
Not sure why ruff and pylint are run on different files? In particular, looks like we're not running ruff on anything in `src/libsemigrous_pybind11/detail` and not running pylint on `docs/source/*.py`??